### PR TITLE
sysvar: add 3 sysvars

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
sysvar: add 3 sysvars

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10566 https://github.com/pingcap/docs/pull/10549
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -588,6 +588,22 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 默认值：`""`
+- 该变量与一个未发布的特性相关。**请勿**设置该变量。
+
+### `tidb_auth_signing_key`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 类型：字符串
+- 默认值：`""`
+- 该变量与一个未发布的特性相关。**请勿**设置该变量。
```
Comment (TomShawn) [inline system-variables.md:605]: 不在 v6.1

File: system-variables.md [hunk 2]
```diff
@@ -2845,6 +2861,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 这个变量是 `transaction_isolation` 的别名。
 
+### `tx_isolation_one_shot`
+
+> **注意：**
+>
+> 该变量用于 TiDB 内部运行，用户不应该修改该变量。
```
Comment (shichun-0415) [inline system-variables.md:2868]: 现有文档这句话有两个相似的版本，咱们或许可以用第一个版本。
> 1. 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
> 2. 该变量是 TiDB 内部使用的变量，**不推荐**修改该变量的值。

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: system-variables.md (hunk 2 above)
Comment (TomShawn) [reply system-variables.md:2868]:
```suggestion
> 该变量仅用于 TiDB 内部实现，不推荐设置该变量。
```

**Pull request merged**